fs.go

```diff
@@ -19,8 +19,25 @@ type tarfs struct {
 	entries map[string]fs.DirEntry
 }
 
+type options struct {
+	disableSeek bool
+}
+
+type Option func(*options)
+
+func DisableSeek(disable bool) func(*options) {
+	return func(o *options) {
+		o.disableSeek = disable
+	}
+}
```

Comment:
I sorta feel like non-seekable files should be the default because it is more performant, but that would be a breaking change for anyone using the library with `http.FS`.

Since this library is at v1, I decided to make this a non-breaking change. But maybe it is worth considering bumping to v2?